modules/ocf_backups/manifests/offsite.pp

```diff
@@ -19,5 +28,25 @@
     '/opt/share/backups/boto.cfg':
       source => 'puppet:///private/boto.cfg',
       mode   => '0600';
+
+    # Box.com credentials and API id/secret
+    '/opt/share/backups/box-creds.json':
+      source => 'puppet:///private/box-creds.json',
+      mode   => '0600';
+
+    # This changes each time a backup is made, so it is not stored on the
+    # puppetmaster and instead is just a local file on the backups server.
+    '/opt/share/backups/box-refresh-token':
+      mode => '0600';
```

Comment:
I think it's fine as long as the job will renew it, but I don't think Puppet shouldn't error on the first run? Does that mean a human will have to manually create the file?